nilearn/reporting/_plot_matrices.py

```diff
@@ -62,6 +62,87 @@ def plot_design_matrix(design_matrix, rescale=True, ax=None, output_file=None):
     return ax
 
 
+def plot_event(model_event, cmap=None, output_file=None, **fig_kwargs):
+    """Creates plot for event visualization.
+
+    Parameters
+    ----------
+    model_event : DataFrame or list of DataFrame
+        the list of Dataframe obtained by ```first_level_models_from_bids```
```

Comment:
```suggestion
        the list of Dataframe obtained for instance by :func:`~nilearn.stats.first_level_model.first_level_models_from_bids`
```
Also, I think that it would be useful to specify what the importance columns of the data-frame are.

Reply:
OK!